frontend/__tests__/src/components/Metrics/MetricsStep/Crews.test.tsx

```diff
@@ -39,57 +39,73 @@ describe('Crew', () => {
   })
 
   it('should selected all options by default when initializing', () => {
-    const { getByText } = setup()
-    const require = getByText('crew A, crew B')
+    const { getByRole } = setup()
 
-    expect(require).toBeInTheDocument()
+    expect(getByRole('button', { name: 'crew A' })).toBeInTheDocument()
+    expect(getByRole('button', { name: 'crew B' })).toBeInTheDocument()
   })
 
   it('should show detail options when click Included crews button', async () => {
     const { getByRole } = setup()
-    await userEvent.click(getByRole('button', { name: mockLabel }))
+    await act(async () => {
+      await userEvent.click(getByRole('combobox', { name: mockLabel }))
+    })
```

Comment:
give `new line` for `given`, `when` and `then`

Reply:
> give `new line` for `given`, `when` and `then`

Nice catch